portfolio/src/main/webapp/info.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+
+<html class="js-focus-visible">
+
+<head>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    
+    <title>Information</title>
+
+    <link rel="stylesheet" href="./css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href="./css/professionStyle.css">
+
+    
+
+</head> 
+
+
+<body data-gr-c-s-loaded="true">
+    <div class="bar">
+        <ul>
+            <li><a href="./index.html" id="onlink">Homepage</a></li>
+            <li><a href="./about.html">About</a></li>
+            <li><a href="./info.html">Info</a></li>
+            <li><a href="https://github.com/nepecg000">GitHub</a></li>
+        </ul>
+    </div>
+    
+    <div class="background"/>
+    <div class="header">Information</div>
+    <div class="horibar"></div>
+
+    <div class="text">
+        <h1>
+            Programming Experience <br><br>
+            <first_project>
```

Comment:
Custom `tags` in HTML are generally discouraged. You can use `div` tags with proper `class` and `id` attributes. So in this example, this can look like - `<div class="project" id="first_project">`